src/components/Card/Card.jsx

```diff
@@ -1,23 +1,113 @@
 import PropTypes from 'prop-types';
 import styles from './Card.module.css'; // gibt .tile als Card_tile__CR1lU aus und greift deshalb nicht mit ${styles.tile}
+import FormInputWithLabel from "../FormInputWithLabel/FormInputWithLabel";
+import {useState} from "react";
 
 export default function Card( {heading, text, list} ) {
 
     const output = text ?
         <p className="mb-0"> {text} </p> :
-        <ul> {list.map((item)=> { return <li key={item}>{item}</li> })} </ul>;
+            <ul className={`px-0 list-group-flush`}>
+                {list.map((item) => { return <li key={item} className={`list-group-item`}>{item}</li> })}
```

Comment:
Das ganze item als key zu übergeben ist nicht so schön. Benutz hier gerne einfach den index wie folgt:
```suggestion
                {list.map((item,index) => { return <li key={index} className={`list-group-item`}>{item}</li> })}
```